ansible/upgrade_sonic.yml

```diff
@@ -72,7 +72,10 @@
           become: true
           shell: sonic_installer install -y {{ image_url }}
           register: output
-          failed_when: output.rc != 0
+          ignore_errors: yes
+
+        - fail: msg="SONiC upgrade failed"
+          when: (output.rc != 0) and (not "'Running SONiC has the same version' in {{ output.stdout }}")
```

Comment:
in this case, I think sonic_installer should return success as well. can you create an issue for sonic_installer for this?

Reply:
OK